models/kyberswap/avalanche_c/kyberswap_avalanche_c_sources.yml

```diff
@@ -1,12 +1,14 @@
 version: 2
 
 sources:
-  - name: kyber_avalanche_c
+  - name: kyber_avalanche_c_trades
     description: "Avalanche (C-Chain) decoded tables related to Kyberswap contract"
     tables:
       - name: DMMPool_evt_Swap
       - name: DMMFactory_evt_PoolCreated
       - name: Elastic_Pool_evt_swap
       - name: Elastic_Factory_evt_PoolCreated
       - name: AggregationRouter_evt_Swapped
+      - name: AggregationRouterV2_evt_Swapped
+      - name: AggregationRouterV3_evt_Swapped
```

Comment:
We may need to remove these two tables from source as well, and add them back when they are available on dune V2 engine.
```suggestion
```